source/components/definition-list/definition-list.less

```diff
@@ -0,0 +1,46 @@
+@import (reference) 'grid/grid';
+hx-dl {
+  display: block;
+}
+
+hx-def {
+  display: flex;
+
+  + hx-def {
+    margin-top: 1.25rem; // 20px
```

Comment:
Given that we want the gap to match the grid, let's use the `@grid-gutter-size` variable instead.

```less
margin-top: @grid-gutter-size;
```